.github/workflows/php.yml

```diff
@@ -10,6 +10,7 @@ jobs:
         with:
           php-version: '7.3'
           extensions: mbstring, intl, gd, xml, dom, json, fileinfo, curl, zip, iconv
+          tools: composer:v1
```

Comment:
Can't we get rid of that now?

Reply:
+1
```suggestion
```

Reply:
Sorry I forgot to clean this commit! It's ok now